events/2022-google-summer-of-code/index.markdown

```diff
@@ -0,0 +1,63 @@
+---
+author: Mark Moll
+comments: false
+date: 2022-02-17 00:00:00 -0700
+layout: page
+slug: 2022-google-summer-of-code
+title: 2022 Google Summer of Code
+---
+# 2022 Google Summer of Code
+
+MoveIt is planning to participate again in the [Google Summer of Code](https://summerofcode.withgoogle.com). Below is a tentative list of project ideas. If you have your own project idea, feel free to create a pull request on the [repository for this page](https://github.com/ros-planning/moveit.ros.org). Unlike previous years, there are now two different project sizes. Medium projects are 175 hours, while large projects are 350 hours. See the [Google Summer of Code FAQ](https://developers.google.com/open-source/gsoc/faq) for details.
+
+## Project ideas
+
+### Python bindings for MoveIt 2
+
+* Use [pybind11](https://pybind11.readthedocs.io/en/latest) to create python bindings for a subset of the MoveIt C++ API.
+* Project size: large
+* Mentor: Henning Kayser
+
+### Improved benchmarking support in MoveIt
+
+* Continue the work from last year's Google Summer of Code project to create more benchmarking capabilities.
```

Comment:
```suggestion
Utilize the [`MoveIt Benchmark Suite`](https://github.com/captain-yoshi/moveit_benchmark_suite) developed during last year's GSoC to prepare a set of realistic scenarios to benchmark independent components of the MoveIt eco-system. Based on these tests, we would like to setup a CI runner the evaluate the performance impact of new pull-requests in MoveIt. An advanced component in this might evaluate the benchmark results for statistically significant changes and point these out in a CI report.
```